ovs-post-process

```diff
@@ -113,10 +118,10 @@ def dpctl_memory_show_process(filename: str):
             else:
                 continue
 
-        finish_samples()
+    finish_samples()
 
 
-def compute_and_log_stats(br_name: str,new_br_data:dict, old_br_data:dict):
+def compute_and_log_stats(br_name: str, new_br_data: dict, old_br_data: dict):
```

Comment:
I'm still such a Python noobie...can you enlighten me on what the `: dict` is doing?  Is it making it require a `dict` for the argument?  If so, what happens if it is not?

Reply:
absolutely nothing happens. its a type annotation so its meant to help IDE tools that I don't use, but other people might.